examples/seismic/accuracy.ipynb

```diff
@@ -0,0 +1,433 @@
+    "        \"\"\"Critical computational time step value.\"\"\"\n",
+    "        return .1"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Verification\n",
+    "\n",
+    "We compute the error between the numerical and reference solutions for varying spatial discretization order and grid spacing. We also compare the time to solution to the error for these parameters."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Discretization order\n",
+    "orders = (2, 4, 6, 8, 10)\n",
+    "norder = len(orders)\n",
+    "\n",
+    "# Domain sizes and gird spacing\n",
+    "shapes = ((201, 2.0), (161, 2.5), (101, 4.0))\n",
+    "dx = [2.0, 2.5, 4.0]\n",
+    "nshapes = len(shapes)\n",
+    "\n",
+    "# Number of time steps\n",
+    "nt = 1501\n",
+    "\n",
+    "# Time axis\n",
+    "\n",
+    "time = np.linspace(0., 150., nt)\n",
+    "# Source peak frequency\n",
+    "f0 = .070"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Fine grid model\n",
+    "c0 = 1.500\n",
+    "model = ModelBench(vp=c0, origin=(0., 0.), spacing=(1.0, 1.0), shape=(401, 401), nbpml=40, dtype=np.float64)\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Source and receiver\n",
+    "src = RickerSource(name='src', grid=model.grid, f0=f0, time=time)\n",
+    "src.coordinates.data[0, :] = np.array(model.domain_size) * .5\n",
+    "src.data[:] = src.data[:] / c0**2\n",
+    "# Define receiver geometry (spread across x, just below surface)\n",
+    "rec = Receiver(name='rec', grid=model.grid, ntime=nt, npoint=1)\n",
+    "rec.coordinates.data[:, 0] = 260.\n",
+    "rec.coordinates.data[:, 1] = 260."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Reference solution for numerical convergence"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 6,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "GNUCompiler: compiled /var/folders/mx/qs0dn9rx7zn6dz2zvwv7tkk00000gn/T/devito-5fqzrgzw/625ad438fd121a644519d4475d5c242bddfd0a8a.c [0.73 s]\n",
+      "=========================================================================================\n",
+      "Section section_1<1499,1> with OI=1.59 computed in 0.001 s [0.26 GFlops/s]\n",
+      "Section section_2<1499,1> with OI=2.48 computed in 0.001 s [0.30 GFlops/s]\n",
+      "Section main<1499,441,441> with OI=6.54 computed in 27.035 s [2.02 GFlops/s, 0.01 GPts/s]\n",
+      "=========================================================================================\n"
+     ]
+    }
+   ],
+   "source": [
+    "solver = AcousticWaveSolver(model, source=src, receiver=rec, time_order=2, space_order=40, dse=None, dle=None)\n",
```

Comment:
why dse/dle disabled?

Reply:
Performance not important there, but maybe not necessary

Reply:
Also, I want some consistency between `space_orders` to compare timings vs error that dse/dle may mess up with

Reply:
it's just to avoid stupid questions from newbies such as "why the hell are dse/dle set to None here"

Reply:
Looks ok without it, removed